assets/pages/contributions-landing/ContributionState.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { usStates, caStates } from 'helpers/internationalisation/country';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+// ----- Types ----- //
+type PropTypes = {
+  countryGroupId: CountryGroupId,
+  selectedState: string,
+};
+
+// ----- Render ----- //
+
+const renderState = (selectedState: string) => ([stateValue, stateName]: [string, string]) => (
+  <option value="{stateValue}" selected={stateValue === selectedState}>{stateName}</option>
+);
+
+const renderStatesField = (selectedState, states) => (
+  <div className="form__field form__field--contribution-state">
+    <label className="form__label" htmlFor="contributionState">State</label>
+    <span className="form__input-with-icon">
+      <select id="contributionState" className="form__input" placeholder="State" required>
+        <option />
+        {(Object.entries(states): any).map(renderState(selectedState))}
```

Comment:
I don't think we have this polyfilled. You could either add it [here](https://github.com/guardian/support-frontend/blob/master/app/views/main.scala.html#L81) or switch to an `Object.keys` implementation.